### ISSUE TITLE
package-lock branches and new dependencies

### ISSUE THREAD

**Issue by andyinno** [cpm-cmake/CPM.cmake#479]
Hello,

As the title says, while working with multiple branches and multiple addition to the project, what I am seeing is that the package-lock.cmake content is updated (I think) with appends of first found written first to file.

This causes the fact that adding different dependencies it found are added in different order. This cause merge branching and rebasing to have a lot of conflicts.

Would be feasible to sort the content of package-lock.cmake in alphabetical order? That should cause less conflicts.

**Comment by TheLartians** [comment]
Good point, it would definitely make sense to sort the package-lock to prevent such conflicts.
